web/ui/react-app/src/Navbar.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Link, LinkGetProps } from "@reach/router";
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  Nav,
+  NavItem,
+  NavLink,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
+
+const isActive = ({ isCurrent }: LinkGetProps) => {
+  return isCurrent ? { style: { color: 'rgba(255,255,255,.5)' } } : null;
+}
+
+const isActiveDropDown = ({ isCurrent }: LinkGetProps) => {
+  return isCurrent ? { style: { backgroundColor: '#343a40', color: '#fff' } } : null;
+}
+
+const ExactNavLink = (props: any) => (
+  <Link getProps={isActive} {...props} />
+);
+
+const DropdownNavLink = (props: any) => (
+  <Link getProps={isActiveDropDown} {...props} />
+);
+
+const Navigation = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggle = () => setIsOpen(!isOpen);
+  return (
+    <Navbar className="mb-3" color="dark" expand="md">  
```

Comment:
This this change, the custom CSS is no longer needed:

```suggestion
    <Navbar className="mb-3" dark color="dark" expand="md">
```